torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1478,7 +1494,28 @@ def _make_tensor_helper(shape, low=None, high=None):
                SkipInfo('TestCommon', 'test_variant_consistency_eager',
                         dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.float16)),),
            sample_inputs_func=sample_inputs_addr),
-
+    OpInfo('amax',
+           op=torch.amax,
+           dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+           dtypesIfCPU=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+           dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+           test_inplace_grad=False,
```

Comment:
`test_inplace_grad` was recently replaced by `supports_inplace_autograd`, but after #54460 would land,
an `OpInfo` for an op should have a `supports_inplace_autograd=False` entry only if it has an inplace variant that does not support autograd.